packages/react-native-renderer/src/__tests__/createReactNativeComponentClass-test.internal.js

```diff
@@ -45,8 +45,12 @@ describe('createReactNativeComponentClass', () => {
 
     expect(Text).not.toBe(View);
 
-    ReactNative.render(<Text />, 1);
-    ReactNative.render(<View />, 1);
+    ReactNative.render(
```

Comment:
Why?

Reply:
Ah- because of the deletions ordering thing. This was a use case (similar to document -> body) that failed. But I think...now it does not fail anymore :) so we can revert this change.